File: go/fizzbuzz.go
package main

import "fmt"

func main() {
  fmt.Println("Hello Go")
  for i := 0; i < 101; i++ {
    if i % 3 == 0 && i % 5 == 0 {
      fmt.Println("FizzBuzz")
    } else if i % 3 == 0 {
      fmt.Println("Fizz")
    } else if i % 5 == 0 {
      fmt.Println("Buzz")
    } else {
      fmt.Println(i)
    }
  }
}

File: clojure/fizzbuzz.clj
(ns fizzbuzz
  (:require [clojure.core.match :refer [match]]))

(doseq [i (range 1 101)]
  (println
   (match [(mod i 3) (mod i 5)]
          [0 0] "FizzBuzz"
          [0 _] "Fizz"
          [_ 0] "Buzz"
          :else i)))
